docs/data/pages.ts

```diff
@@ -169,6 +169,25 @@ const pages: MuiPage[] = [
         title: 'Time range picker 🚧',
         plan: 'pro',
       },
+      {
+        pathname: '/x/react-date-pickers/fields',
+        title: 'Field components',
+        scopePathnames: [
+          '/x/react-date-pickers/fields',
+          '/x/react-date-pickers/date-field',
+          '/x/react-date-pickers/date-range',
+          '/x/react-date-pickers/date-time-field',
+          '/x/react-date-pickers/time-field',
+        ],
+        newFeature: true,
+        children: [
+          { pathname: '/x/react-date-pickers/fields', title: 'Getting started' },
+          { pathname: '/x/react-date-pickers/date-field' },
+          { pathname: '/x/react-date-pickers/date-range-field', plan: 'pro' },
+          { pathname: '/x/react-date-pickers/date-time-field' },
+          { pathname: '/x/react-date-pickers/time-field' },
```

Comment:
For me it makes more sense to group the docs by community first, pro second such that it becomes easier to distinguish free and paid components.

And sort than by date/time/dateTime because it's like two atoms and their combination
```suggestion
          { pathname: '/x/react-date-pickers/time-field' },
          { pathname: '/x/react-date-pickers/date-time-field' },
          { pathname: '/x/react-date-pickers/date-range-field', plan: 'pro' },
```

Reply:
Great observation, I would agree with this suggestion. 👍 
DateField -> TimeField -> DateTimeField -> DateRangeField -> ...

Reply:
Fine for me
As long as we keep the same order everywhere

Reply:
Done